indico/modules/events/templates/display/administrative/event.html

```diff
@@ -0,0 +1,89 @@
+{% from 'events/display/common/_legacy.html' import render_event_time, render_location %}
+{% from 'events/display/common/_manage_button.html' import render_manage_button %}
+
+<table class="eventWrapper">
+    <tr>
+        <td>
+            <div class="manage-button-container">
+                {{ render_manage_button(event, 'EVENT', toggle_notes=false, show_note_operations=true) }}
+            </div>
+            <div class="reportNumberHeader">
+                {% for ref in event.references %}
+                    {{ ref.value }}
+                    {% if not loop.last %}
+                        <br>
+                    {% endif %}
+                {% endfor %}
+            </div>
+            {% block header %}
+                <br>&nbsp;<br>
+                <div class="eventHeader">
+                    {{ theme_settings.title_fr or
+                       'ORGANISATION EUROP&Eacute;ENNE POUR LA RECHERCHE NUCL&Eacute;AIRE' | safe }}
+                    <br>
+                    <span class="CERNTitle"> CERN </span>
+                    {{ theme_settings.title_en or
+                       'EUROPEAN ORGANIZATION FOR NUCLEAR RESEARCH' | safe }}
+                </div>
+                <div align="center">
+                    <hr width="50%">
+                </div>
+            {% endblock %}
+
+            <br>&nbsp;<br>
+
+            <div class="eventInfo">
+                {{ event.title }}
+                <br>
+                {% block location_time %}
+                    {% if event.has_location_info %}
+                        {{ render_location(event) }}
+                    {% endif %}
+                    {{ render_event_time(event, timezone=timezone) }}
+                {% endblock %}
+                <br>
+                {% block material %}
+                {% endblock %}
```

Comment:
i'd move that to the previous line so it's immediately clear that the block is empty